extension/solarwindsapmsettingsextension/extension.go

```diff
@@ -25,13 +32,63 @@ func newSolarwindsApmSettingsExtension(extensionCfg *Config, logger *zap.Logger)
 	return settingsExtension, nil
 }
 
-func (extension *solarwindsapmSettingsExtension) Start(_ context.Context, _ component.Host) error {
+func (extension *solarwindsapmSettingsExtension) Start(ctx context.Context, _ component.Host) error {
 	extension.logger.Debug("Starting up solarwinds apm settings extension")
-	_, extension.cancel = context.WithCancel(context.Background())
+	ctx, extension.cancel = context.WithCancel(ctx)
+	configOk := validateSolarwindsApmSettingsExtensionConfiguration(extension.config, extension.logger)
+	if configOk {
+		extension.conn, _ = grpc.Dial(extension.config.Endpoint, grpc.WithTransportCredentials(credentials.NewTLS(&tls.Config{})))
+		extension.logger.Info("Dailed to " + extension.config.Endpoint)
+		extension.client = collectorpb.NewTraceCollectorClient(extension.conn)
+		go func() {
+			ticker := newTicker(extension.config.Interval)
+			defer ticker.Stop()
+			for {
+				select {
+				case <-ticker.C:
+					refresh(extension)
+				case <-ctx.Done():
+					extension.logger.Debug("Received ctx.Done() from ticker")
+					return
+				}
+			}
+		}()
+	} else {
+		extension.logger.Warn("solarwindsapmsettingsextension is in noop. Please check config")
+	}
```

Comment:
This is not needed. `Config.Validate()` is called by the builder automatically and collector doesn't start with returned error message

Reply:
Yes, but it can break user's application when it is used in collector lambda layer.

Reply:
Sure. I will raise that to lambda project.

Reply:
Raised a [feature request issue](https://github.com/open-telemetry/opentelemetry-lambda/issues/1181)

Reply:
@MovieStoreGuy We are thinking to add back `Config.Validate()` and replace any invalid field of the configuration by default values. As we set default values, `Config.validate()` will always return `nil`. And the extension will not need to work in noop mode anymore. 

What do you think?

Reply:
Added back `Config.Validate()`